rules_chibios.mk

```diff
@@ -0,0 +1,28 @@
+CFLAGS   += -D__MAVRIC_ENDIAN_LITTLE__
+CXXFLAGS += -D__MAVRIC_ENDIAN_LITTLE__
+
+
+LIB_SRCS += hal/chibios/gpio_chibios.cpp
+LIB_SRCS += hal/chibios/i2c_chibios.cpp
+LIB_SRCS += hal/chibios/pwm_chibios.cpp
+LIB_SRCS += hal/chibios/serial_chibios.cpp
+LIB_SRCS += hal/chibios/spi_chibios.cpp
+LIB_SRCS += hal/chibios/stubs.cpp
+LIB_SRCS += hal/chibios/time_keeper.cpp
+
+# CSRC += hal/chibios/usbcfg.c
+
+
+# LIB_SRCS += boards/mavrimini.cpp
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/326/commits/52c933f564875f068b4c38168604e9406bd3fdcf